README.md

```diff
@@ -323,7 +292,7 @@ cat generated/logs/example_team/stationkeeping/0/trial_score.txt
 ### Generating a single trial video
 
 After running a trial, a `state.log` file is stored under `generated/logs/<team>/<task>/<trial_num>/gazebo-server`. This is a playback log file that allows you to play back the trial. 
-To generate a trial video, please run the trial using the steps above, source vrx, and then run
+To generate a trial video, please run the trial using the steps above, source vorc, and then run
 
 ```
 ./generate_trial_video.bash example_team stationkeeping 0
```

Comment:
Should the team be `ghostship` here?

Reply:
Yeah I should replace all the `example_team` references and in fact remove the two team folders from vrx.
@crvogt how much work is it to create a second solution Docker image, just so that we have more than one team to test the multi-scripts? It could just be something trivial again, perhaps the boat moving backwards, i.e. "ghostship is back"...... or something more creative.

Reply:
String replaced in a086d03